### ISSUE TITLE
 Labial adhesion

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6650]
**Preferred term label:**
 Labial adhesion

**Synonyms**
Synechia vulvae, labial agglutination

**Definition (free text, please give PubMed ID)**
Fusion of the labia minora or majora.
PMID: 29262197

**Parent term (use hpo.jax.org/app)**
Abnormality of the labia

**Comment by pnrobinson** [comment]
added term
